meta-client/src/test/java/org/apache/helix/metaclient/impl/zk/TestZkMetaClient.java

```diff
@@ -84,6 +89,40 @@ public void testCreateContainer() {
     }
   }
 
+  @Test
+  public void testCreateTTL() {
+    final String key = "/TestZkMetaClient_testTTL";
+    try (ZkMetaClient<String> zkMetaClient = createZkMetaClient()) {
+      zkMetaClient.connect();
+      zkMetaClient.createWithTTL(key, ENTRY_STRING_VALUE, 1000);
+      Assert.assertNotNull(zkMetaClient.exists(key));
+    }
+  }
+
+  @Test
+  public void testRenewTTL() {
+    final String key = "/TestZkMetaClient_testRenewTTL_1";
+    try (ZkMetaClient<String> zkMetaClient = createZkMetaClient()) {
+      zkMetaClient.connect();
+      zkMetaClient.createWithTTL(key, ENTRY_STRING_VALUE, 10000);
+      Assert.assertNotNull(zkMetaClient.exists(key));
+
+      MetaClientInterface.Stat stat = zkMetaClient.exists(key);
+
+      // Sleep for a small period of time for a bigger difference between
+      // creation and modified time.
+      Thread.sleep(5000);
```

Comment:
Is this necessary for testing?

Reply:
Hmm it was there when I was printing statements but I guess it isn't necessary, will remove them. Thanks!